### PR TITLE
rename farmers to storagenode

### PR DESCRIPTION
**Pull request by aligeti**
rename farmers to storagenode

---
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/275)

### CODE REVIEW

File: cmd/farmer/main.go [hunk 1]
```diff
@@ -18,12 +18,12 @@ import (
 
 var (
 	rootCmd = &cobra.Command{
-		Use:   "farmer",
-		Short: "Farmer",
+		Use:   "storagenode",
```
Comment (jtolds) [inline cmd/farmer/main.go]: should the filename of this file change?

File: cmd/farmer/piecestore-farmer.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 60
       containers:
-      - image: "storjlabs/piecestore-farmer"
+      - image: "storjlabs/piecestore-storagenode"
```
Comment (jtolds) [inline cmd/farmer/piecestore-farmer.yaml]: also the filename of this file. the image name here can probably just be storjlabs/storagenode

File: cmd/farmer/main.go (hunk 1 above)
Comment (aligeti) [reply cmd/farmer/main.go]: it shows as 

rootCmd = &cobra.Command{
		Use:   "storagenode",
		Short: "StorageNode",
	}

in the PR branch. Can you refresh and check again please.

File: cmd/farmer/piecestore-farmer.yaml (hunk 2 above)
Comment (aligeti) [reply cmd/farmer/piecestore-farmer.yaml]: I renamed the farmer folder in cmd/farmer/piecestore-farmer.yaml -> cmd/storagenode/piecestore-farmer.yaml & also the renamed 
piecestore-farmer.yaml -> storagenode.yaml

**Pull request merged**